### PR TITLE
feat: append more definitions to db.json

### LINKED ISSUE #7: [FEATURE] Include a few more abbreviations
### Description

Consider appending the abbreviations on the homepage to the JSON database 

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by slim-codes**
## Fixes Issue

Closes #7 

## Changes proposed
- These changes append a few more abbreviations to the JSON database (particularly those displayed on the homepage)

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot_20221003-094818_1

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -11,5 +11,58 @@
...
+        "definition": "I've Got To Go"
+    },
+
+    "icymi":{
+        "definition": "In Case You Missed It"
+    },
+
+    "kiss":{
+        "definition": "Keep It Simple Stupid",
+        "alternatives": "keep it super simple, keep it short and simple"
```
Comment (Njong392) [inline public/server/db.json:62]: Each letter of `alternatives` should begin with a capital letter, just as with `definitions`

```suggestion
        "alternatives": "Keep It Super Simple, Keep It Short And Simple"
```

Comment (Njong392) [inline public/server/db.json:62]: Each letter of `alternatives` should begin with a capital letter, just as with `definitions`
```suggestion
        "alternatives": "Keep It Super Simple, Keep It Short And Simple"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: public/server/db.json (hunk 1 above)
Comment (slim-codes) [reply public/server/db.json]: Must have been an oversight, good catch

**Comment by slim-codes** [comment]
You're welcome

File: public/server/db.json [hunk 2]
```diff
@@ -11,5 +11,58 @@
 
     "idk":{
         "definition": "I Don't Know"
+    },
+
+    "imo":{
+        "definition": "I My Opinion"
```
Comment (Olanetsoft) [inline public/server/db.json:17]: @slim-codes I think there is a typo here. It should be `In my opinion`

Comment (slim-codes) [reply public/server/db.json]: @Olanetsoft Nice catch. Thanks for that. I've fixed it

**Review by slim-codes — COMMENTED** [review]
Typo Fixed

**Comment by Njong392** [comment]
Thank you @Olanetsoft and @slim-codes for the awesome collaboration on this PR. Merged! :tada:

**Pull request merged**